src/main/java/org/jabref/gui/preferences/preview/PreviewTabViewModel.java

```diff
@@ -304,6 +316,7 @@ public void selectedInChosenDown() {
 
     public void resetDefaultLayout() {
         PreviewLayout defaultLayout = findLayoutByName("Preview");
+        System.out.println("resetDefaultLayout");
```

Comment:
Please use our logger infrastructure.

Reply:
sorry, just for debug. I will remove it.